src/editor/lib/iconsets/wp-editor/wp-editor.js

```diff
@@ -276,6 +276,9 @@ const template = html`
 <g id="fa-fill" viewBox="0 0 512 512">
   <path d="M502.63 217.06L294.94 9.37C288.69 3.12 280.5 0 272.31 0s-16.38 3.12-22.62 9.37l-81.58 81.58L81.93 4.77c-6.24-6.25-16.38-6.25-22.62 0L36.69 27.38c-6.24 6.25-6.24 16.38 0 22.63l86.19 86.18-94.76 94.76c-37.49 37.49-37.49 98.26 0 135.75l117.19 117.19c18.75 18.74 43.31 28.12 67.87 28.12 24.57 0 49.13-9.37 67.88-28.12l221.57-221.57c12.49-12.5 12.49-32.76 0-45.26zm-116.22 70.97H65.93c1.36-3.84 3.57-7.98 7.43-11.83l13.15-13.15 81.61-81.61 58.61 58.6c12.49 12.49 32.75 12.49 45.24 0 12.49-12.49 12.49-32.75 0-45.24l-58.61-58.6 58.95-58.95 162.45 162.44-48.35 48.34z"/>
 </g>
+<g id="fa-circle-exclamation" viewBox="0 0 512 512">
```

Comment:
1. this is a dynamically generated file. your change will be overwritten the next time the [icon set is regenerated](https://github.com/UCDavisLibrary/cork-icon-set-generator).
2. This icon should already be loaded in the editor side, so no need to add it.